ckanext/versioning/tests/test_helpers.py

```diff
@@ -47,3 +48,19 @@ def test_dataset_does_not_has_link_resources(self):
         assert_equals(
             helpers.has_link_resources(self.dataset),
             False)
+
+    def test_get_dataset_revision_list(self):
```

Comment:
Why is this test in a PR regarding package_show? It seems like another piece of code ... 😉